src/Ghcid.hs

```diff
@@ -33,7 +34,6 @@ import Language.Haskell.Ghcid.Util
 import Language.Haskell.Ghcid.Types
 import Wait
 
-import Data.Functor
```

Comment:
Old GHCs might not like this

Reply:
I have plenty of CI tests for that kind of thing :)

Reply:
My CI will know :)